python-main.js

```diff
@@ -280,6 +281,20 @@ function translations() {
         };
     }
 
+    /* Checks URL for query parameter 'l' and translates page to that language. */
+    function setLang(message){
+        if (message.l){
+            var lang = message.l;
+            if (validLangs.indexOf(lang)>-1){
+                updateLang(lang, function(translations) {
+                    config.translate = translations;
```

Comment:
`config` is an argument to `web_editor()`, so normally this object shouldn't have been able to see it here.
However, thanks to the wonderful weirdness of JS scopes and context, due to where this function is called from it has `config` in scope.

For this type of implementation a method called from `web_editor()` should also ask from a callback as a parameter, so that web_editor could do something like:

```js
TRANSLATIONS.setLang(qs, function(translations) {
    config.translate = translations;
    ...
}); 
```